### ISSUE TITLE
Confusing text in EUF-CMA section

### ISSUE THREAD

**Issue by danvangeest** [lamps-wg/draft-composite-sigs#77]
This sentence in section 11.1 is confusing, in particular the part in bold.

   The latter
   version bears a resemblance to a stripping attack, which parallel
   signatures are subject to, but is slightly different in that the
   cross-protocol EUF-CMA game **also considers modification message
   definition as signed differs from the message the verifier accepts**.

I don't even have a suggestion on how to fix that, so I'll leave it to the authors.

Also I suggest this text to fix the next sentence:

   In contrast, stripping attacks consider removing one component signature and attempting to verify the remaining signature with the
   same message.

**Comment by ounsworth** [comment]
Yup, totally agree. That text was horrible. I think it was just flavour text and not particularly important to the EUF-CMA point, so I just removed it:

f9658ba306569d866ec812723aacfdc3fec81056
